package.json

```diff
@@ -13,8 +13,9 @@
   "scripts": {
     "dev-up": "npm install && (cd app && npm install) && npm run build",
     "test": "gulp test",
+    "test-watch": "mocha --watch --require babel-register --recursive ./test",
```

Comment:
Can't really remember how gulp works >< Is this related to the `tdd` gulp task above?